packages/accordions/src/styled/accordion/StyledInnerPanel.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'accordions.step_inner_panel';
+
+export interface IStyledInnerPanel {
+  isExpanded?: boolean;
+}
+
+export const StyledInnerPanel = styled.div.attrs<IStyledInnerPanel>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledInnerPanel>`
+  transition: max-height 0.25s ease-in-out;
+  /* stylelint-disable-next-line declaration-no-important */
+  max-height: ${props => !props.isExpanded && '0 !important'};
```

Comment:
"...and document why the `!important` is necessary". Aka, add a `/* [1] */` with doc added to the header.

https://github.com/zendeskgarden/react-components/blob/348329cfed710b389be93ce5aa0d26f5edb59d5a/packages/dropdowns/src/styled/menu/StyledMenu.ts#L31-L32

Reply:
> "...and document why the !important is necessary"

Yeah I noticed this comment. I went and looked at the codebase, and places where I saw `!important` used, no comments were added, which sort of confused me. I'll go ahead an add a comment here.

Reply:
I've linked to a location ☝️ that shows exactly what I'm looking for.

Reply:
Awesome, thanks 👍 .